src/messages.cpp

```diff
@@ -141,59 +137,30 @@ namespace quicr::messages {
         buffer << UintVar(msg.last_group_id);
         buffer << UintVar(msg.last_object_id);
 
+        UintVar size(buffer.size());
+        buffer.insert(buffer.begin(), size.begin(), size.end());
+
```

Comment:
wouldn't this causes reshuffling of the elements ?

Reply:
Yep! Thank UintVar for that.

Reply:
Would have been nice to be fixed value instead of uintvar for header length.  @suhasHere , do you have another way to do it?